content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -340,12 +340,8 @@ namespaces based on their labels.
 
 ## Targeting a Namespace by its name
 
-{{< feature-state for_k8s_version="1.22" state="stable" >}}
```

Comment:
I'd have left this in, _but_ it's also OK to remove it (it's just barely old enough).

Reply:
Thanks @sftim for so much!